src/main/java/com/google/collegeplanner/servlets/PlannerPageServlet.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.collegeplanner.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+
+/** Servlet that renders the /planner page.*/
+@WebServlet("/planner")
+public class PlannerPageServlet extends HttpServlet {
+  /**
+   * Renders planner.jsp
```

Comment:
Nitpicking, but for single line comments, instead of using typing out
/*
* Renders planner.jsp
*/

you can just write
// Renders planner.jsp

Reply:
it's just for consistency. all of our functions have the multi line comment syntax no matter how long they are

Reply:
Ok cool